src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,42 @@
+.bleedWrapper {
+  background: var(--color-gray-300);
```

Comment:
You can't apply this background color to this component, since it ends up in the bleed areas. In the designs, the gray background is only supposed to be as wide as the main 12 columns.

Reply:
👍 Understood. The gray background is displayed behind the content with the padding now.